src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Completion/RazorCompletionEndpointTest.cs

```diff
@@ -15,21 +15,14 @@ namespace Microsoft.AspNetCore.Razor.LanguageServer.Completion
 {
     public class RazorCompletionEndpointTest : LanguageServerTestBase
     {
-        public RazorCompletionEndpointTest()
-        {
-            CompletionListProvider = new AggregateCompletionListProvider(Array.Empty<CompletionListProvider>());
-        }
-
-        private AggregateCompletionListProvider CompletionListProvider { get; }
-
         [Fact]
         public async Task Handle_NoDocumentContext_NoCompletionItems()
         {
             // Arrange
             var documentPath = "C:/path/to/document.cshtml";
             var documentContextFactory = new TestDocumentContextFactory();
             var completionEndpoint = new RazorCompletionEndpoint(
-                documentContextFactory, CompletionListProvider, LoggerFactory);
+                documentContextFactory, null, LoggerFactory);
```

Comment:
Nit: I prefer to label null parameters so we know what we're excluding.

Reply:
I do too... except in tests where I just want the compiler to shut up 😛

Will make this better